src/app/modals/share-link-modal/share-link-modal.component.scss

```diff
@@ -16,15 +16,15 @@
     font-stretch: normal;
     line-height: 18px;
     padding-top: 10px;
-} 
+}
 .value{
     font-size: 13px;
     word-wrap: break-word;
     padding: 5px 0;
 }
 .copy-clipboard{
     font-size: 12px;
-    color: #a4a4a4;
+    color: var(--blue);
     border-bottom: 1px dotted #ddd;
```

Comment:
I see in the comment of the ticket, Nick says "link color and underline var(--blue)". Does that mean we also want to make the `border-bottom` blue?

Reply:
Ack, you are right! Done.